src/stores/orderStore.ts

```diff
@@ -51,92 +52,101 @@ class OrderStore extends BaseStore {
   placeOrder = async (orderType: string, body: any) => {
     switch (orderType) {
       case OrderType.Market:
-        return this.api
-          .placeMarket(body)
-          .then(this.orderPlacedSuccessfully, this.orderPlacedUnsuccessfully)
-          .then(() => Promise.resolve());
+        return (
+          this.api
+            .placeMarket(body)
+            // tslint:disable-next-line:no-empty
+            .then(() => {}, this.orderPlacedUnsuccessfully)
+            .then(() => Promise.resolve())
+        );
       case OrderType.Limit:
-        return this.api
-          .placeLimit(body)
-          .then(this.orderPlacedSuccessfully, this.orderPlacedUnsuccessfully)
-          .then(() => Promise.resolve());
+        return (
+          this.api
+            .placeLimit(body)
+            // tslint:disable-next-line:no-empty
+            .then(() => {}, this.orderPlacedUnsuccessfully)
+            .then(() => Promise.resolve())
+        );
     }
   };
 
   editOrder = async (body: any, id: string) =>
     this.api
       .cancelOrder(id)
       .then(() => this.api.placeLimit(body))
-      .then(this.updateOrders)
-      .then(this.orderEditedSuccessfully)
       .catch(this.orderPlacedUnsuccessfully);
 
   cancelOrder = async (id: string) => {
     await this.api.cancelOrder(id);
-    this.notificationStore.addNotification(
-      levels.information,
-      `${messages.orderCancelled} ${id}`
-    );
-    this.updateOrders();
   };
 
   cancelAll = async () => {
-    const orders = this.rootStore.orderListStore.limitOrders;
-    const promises = orders.map((order: OrderModel) => {
-      return this.api.cancelOrder(order.id);
+    this.rootStore.orderListStore.limitOrders.forEach((order: OrderModel) => {
```

Comment:
`this.rootStore.orderListStore.limitOrders.map(o=>o.id).forEach(this.api.cancelOrder)` would be more readable i believe